src/BlazorWebFormsComponents/LoginControls/Login.razor.cs

```diff
@@ -36,16 +36,16 @@ public partial class Login : BaseWebFormsComponent
 		[Parameter] public string LoginButtonImageUrl { get; set; }
 		[Parameter] public string LoginButtonText { get; set; } = "Log In";
 		[Parameter] public ButtonType LoginButtonType { get; set; } = ButtonType.Button;
-
+		[Parameter] public string Password { get => Model?.Password ?? string.Empty; set => Model.Password = value; }
```

Comment:
`Password` should be get, but due the lack of Blazor parameter I made it getter & setter